thunder/core/jit_ext.py

```diff
@@ -1254,8 +1254,8 @@ def from_opaque(provenance, *, new_output=False):
             raise NotImplementedError(f"unpacking from OPAQUE {fn.value} {provenance}")
 
         def from_provenance(provenance, *, new_output=False):
-            if hasattr(provenance, "proxy"):
-                return provenance.proxy  # bind?
+            if (p := getattr(provenance, "proxy", None)) is not None:
```

Comment:
Nit: Can we not use := just to save a vertical line of code, please? It's cool in elif changes, but here...

Reply:
Sure. I saw other uses of `:=` like this in this repo, but this is not really necessary.